index.html

```diff
@@ -0,0 +1,37 @@
+
+
+<!DOCTYPE html>
+<html>
+<head>
+   <title>Grow A Tree</title>
+</head>
+<body>
+   <h1>Trees Are Great.</h1>
+   <form action="#" method="post">
```

Comment:
THANK YOU! This is such helpful feedback. I suspect it explains why I was having difficulty adding some more conditionals to the ```Enter``` key press.